receiver/prometheusreceiver/internal/metrics_adjuster.go

```diff
@@ -249,11 +257,12 @@ func (ma *MetricsAdjuster) adjustMetricTimeseries(metric *metricspb.Metric) bool
 				// reset timeseries
 				tsi.initial = current
 				tsi.previous = current
+				dropped++
```

Comment:
Q.: is it worth to distinguish between init and reset?

Reply:
If this was a general property of receivers it would be but it's specific to Prometheus so I don't think it's something to expose directly in the receiver metrics.

We might want to consider adding a tag for dropped timeseries so that different receivers can add annotations about why timeseries got dropped.

Reply:
SG